itest/rust/src/object_tests/object_test.rs

```diff
@@ -1076,3 +1076,59 @@ fn double_use_reference() {
     double_use.free();
     emitter.free();
 }
+
+// ----------------------------------------------------------------------------------------------------------------------------------------------
+
+#[derive(GodotClass)]
+#[class(init, base=Object)]
```

Comment:
Could you add a simple `impl IObject for MultipleImplBlocks { fn ready() { ... } }` to avoid regressions related to that.

Reply:
to what end?

Reply:
To catch if a future change causes some failure at the confluence of multiple inherent impls and a trait impl.  Could even be an empty impl just to invite the compiler error if it should come.

Reply:
I added a dummy init

```rust
#[godot_api]
impl IObject for MultipleImplBlocks {
    fn init(_base: Base<Self::Base>) -> Self {
        Self {}
    }
}
```